executor/executor.go

```diff
@@ -175,6 +175,17 @@ type Executor interface {
 	newFirstChunk() *chunk.Chunk
 }
 
+// Next is a wrapper function on e.Next(), it handles some common codes.
+func Next(ctx context.Context, e Executor, req *chunk.RecordBatch) error {
```

Comment:
Do we need to export this function?

Reply:
It's better to export this function.
As `e.Next()` is replaced by `Next()` now, calling `e.Next()` doesn't reflect the truth.
We may call `execute.Next(e)` outside the execute package, for example, writing tests.